content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -30,13 +30,14 @@ Starting from release 10.17, Core, UI, and several other components such as Mess
 
 ##### Breaking change in the Alarms, Events, Measurements APIs - required parameters will be introduced
 
-As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), in a future release, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
+As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), in the future, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
```

Comment:
```suggestion
As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), in a future version, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
```